src/types/api.js

```diff
@@ -120,6 +122,8 @@
  * @prop {boolean} isScopedToUri
  * @prop {string} name
  * @prop {boolean} canLeave
+ * @prop {Object} links
+ *   @prop {string} links.html
```

Comment:
This link _is_ a property on an API object, so it should be listed higher up with the other properties that are part of the API object.